tests/kernel/mellanox_config.pm

```diff
@@ -37,8 +37,9 @@ sub run {
     my $protocol = get_var('MLX_PROTOCOL') || 2;
 
     if (is_sle('>=15')) {
-        my $GA_REPO = 'http://download.suse.de/ibs/SUSE:/SLE-15:/GA/standard/SUSE:SLE-15:GA.repo';
-        zypper_call("ar -f -G $GA_REPO");
+        zypper_call("ar -f -G http://download.suse.de/ibs/SUSE:/SLE-15-SP1:/GA:/TEST/images/repo/SLE-15-SP1-Module-Development-Tools-POOL-x86_64-Media1/  dev_1");
```

Comment:
Is it a good idea to hard-code the URLs for SLE15 SP1? For the next version you'll need to touch this again.

Reply:
@asmorodskyi proposed so... you are right, but I might create another condition for SP2 ...

Reply:
I also recommend to strongly reconsider this version hardcoding. Yes, might be that we need to change it again but I suggest to start under the assumption that you will *not* need changes. I will merge the PR now nevertheless but this is something to consider for a followup task